app/components/flash_message_component.html.erb

```diff
@@ -3,6 +3,6 @@
   classes: classes,
   html_attributes: { role: role },
 ) do |notification_banner| %>
-  <%= notification_banner.slot(:heading, text: heading) %>
+  <%= notification_banner.heading(text: heading) %>
```

Comment:
Getting some dupes in here

<img width="1018" alt="Screenshot 2021-06-09 at 16 42 11" src="https://user-images.githubusercontent.com/642279/121386214-b7e18700-c941-11eb-8d99-500c4a65fca8.png">

<img width="1044" alt="Screenshot 2021-06-09 at 16 41 48" src="https://user-images.githubusercontent.com/642279/121386250-bfa12b80-c941-11eb-8531-6094c74e6e78.png">

Reply:
```suggestion
  <% notification_banner.heading(text: heading) %>
```
🙊

Reply:
[Looking at how this works](https://github.com/DFE-Digital/govuk-components/blob/version-2.0.0/app/components/govuk_component/notification_banner_component.html.erb#L8), because it's a capturing block the `#heading` is both added in the conventional manner _and_ then included in the `content`. I can see this catching other people out, perhaps to inject ad-hoc content there should be another slot (`body`?) to make it more obvious what's going on.

Reply:
Yeah, I’m never sure when to use `<%` and `<%=` when using the component slots.